.env.example

```
@@ -5,6 +5,7 @@ DATABASE_URL="postgresql://user:password@localhost:5432/survey_platform_dev"
 REDIS_URL="redis://localhost:6379"
 
 # Authentication (Clerk) - Get keys from https://clerk.dev
+CLERK_PUBLISHABLE_KEY="pk_test_your_key_here"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)